index.html

```diff
@@ -53,6 +53,230 @@ <h1>I'm Anold. <br>Glad to see <br> you!</h1>
+              <li>
+                <div class="tech_container">
+                  <h6>Languages</h6>
+                  <div class="tech_img">
+                    <img src="images/Disabled.png" alt="arrow down icon">
+                  </div>
+                </div>
+                <ul class="technologies_container-languages">
+                  <!--languages-->
+                  <li>
+                    <div>
+                      <img src="images/js icon.png" alt="javascript icon">
+                    </div>
+                    <div>
+                      <p>JavaScript</p>
+                    </div>
+                  </li>
+                  <li>
+                    <div>
+                      <img src="images/html icon.png" alt="html icon">
+                    </div>
+                    <div>
+                      <p>HTML</p>
+                    </div>
+                  </li>
+                  <li>
+                    <div>
+                      <img src="images/css icon.png" alt="css icon">
+                    </div>
+                    <div>
+                      <p>CSS</p>
+                    </div>
+                  </li>
+                </ul>
+              </li>
+
+              <li>
+                <div class="tech_container border_bottom ">
+                  <h6>Frameworks</h6>
+                  <div class="tech_img">
+                    <img src="images/Icon - Chevron - Right - Larger.png" alt="arrow right icon">
+                  </div>
+                </div>
+              </li>
+
+              <li>
+                <div class="tech_container border_bottom ">
+                  <h6>Skills</h6>
+                  <div class="tech_img">
+                    <img src="images/Icon - Chevron - Right - Larger.png" alt="arrow right icon">
+                  </div>
+                </div>
+              </li>
+            </ul>
+          </div>
+        </article>
+      </article>
+
+        <article class="contact_me">
+          <H2>Contact me</H2>
+          <p>If you have an application you are interested in developing, a feature that you need built or a project
+            that needs
+            coding.
+            I'd love to help with it!</p>
+          <form action="">
+            <fieldset class="form">
+              <label for="name">
+                <input type="text" id="name" placeholder="Lorem ipsum">
+              </label>
+              <label for="email">
+                <input type="email" id="email" placeholder="loremipsum@gmail.com">
+              </label>
+              <textarea name="message" id="message" cols="30" rows="10" placeholder="write your message here">
+              </textarea>
+              <br>
+              <input class="form_btn" type="submit" name="submit" id="submit" value="Submit">
+            </fieldset>
+          </form>
+          <div class="divider"></div>
+        </article>
+       
+    </section>
```

Comment:
I think that since this section is not part of this milestone, you could kindly copy it to a branch to be submitted in the future so it can be checked properly for that milestone 😉